### PR TITLE
MXP-1034: free cancellation networking layer

### PR DESCRIPTION
**Pull request by alexis-miakouikila**

### CODE REVIEW

File: packages/demand-api/src/trip/types.ts [hunk 1]
```diff
@@ -210,6 +210,36 @@ export type GetTripPositionResponse = {
...
+
+  // ---------------------------
+  // Case2: no cancellation fee
+  // ---------------------------
+  // {
+  // "cancellation_fee":false
+  // }
+  // ---------------------------
+}
+
```
Comment (debugme) [inline packages/demand-api/src/trip/types.ts:242]: Perhaps we can delete this big commented out block?

File: packages/demand-bloc-quotes/src/QuotesBloc.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  QuotesV2,
+  QuotesV2Response,
+  QuotesV2SearchParams,
 } from '@karhoo/demand-api'
 import { Subject, Subscription, timer } from 'rxjs'
-import { publishReplay, refCount, map, distinctUntilChanged } from 'rxjs/operators'
+import { distinctUntilChanged, map, publishReplay, refCount } from 'rxjs/operators'
 import { poll } from './polling'
-import { transformer, QuoteItem } from './transformer'
+import { QuoteItem, transformer } from './transformer'
```
Comment (debugme) [inline packages/demand-bloc-quotes/src/QuotesBloc.ts:11]: How come the order of these imports has changed? Are you using different prettifier rules?

Comment (alexis-miakouikila) [reply packages/demand-bloc-quotes/src/QuotesBloc.ts:11]: Truly I have no idea 🤷 
I will revert that back.

Comment (alexis-miakouikila) [reply packages/demand-bloc-quotes/src/QuotesBloc.ts:11]: I haven't touched the linting rules. I checked my vscode extension but could not see anything relevant. I try changing it but it comes back as soon as.I press save. I could use another IDE or something like vim, but I don't really see the point.

File: packages/demand-bloc-quotes/src/QuotesBloc.ts [hunk 3]
```diff
@@ -198,7 +198,6 @@ export class QuotesBloc {
     const handleQuotesLoaded = (res: HttpResponse<QuotesV2Response>) => {
       if (res.ok) {
         const items = res.body?.quotes || []
-
```
Comment (Nick-Zag) [inline packages/demand-bloc-quotes/src/QuotesBloc.ts:201]: return  whitespace please)

**Review by Nick-Zag — CHANGES_REQUESTED** [review]

**Review by Nick-Zag — APPROVED** [review]

**Pull request merged**